mysql/changelog.d/17002.fixed

```diff
@@ -0,0 +1 @@
+[MySQL] - SDBM-830 - Make the autocommit explicit
```

Comment:
```suggestion
Explicitly activate autocommit mode for monitoring connections
```
I was told not include jira tickets in customer facing changelog. plus integration changelog are scoped to the integration so no need to add [MySQL].